### PR TITLE
feat: add some more job metadata to opentelemetry

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

File: jobserver/api/jobs.py [hunk 1]
```diff
@@ -127,8 +129,12 @@ def post(self, request, *args, **kwargs):
                             job.status not in completed
                             and job_data["status"] in completed
                         )
+
                         span.set_attribute("previous_status", job.status)
                         span.set_attribute("status", job_data["status"])
+                        span.set_attribute("started_at", job.started_at)
```
Comment (madwort) [inline jobserver/api/jobs.py:135]: this is wrong!!!

File: jobserver/api/jobs.py [hunk 2]
```diff
@@ -118,6 +118,8 @@ def post(self, request, *args, **kwargs):
                     )
 
                     span.set_attribute("job_id", str(job.id))
```
Comment (bloodearnest) [inline jobserver/api/jobs.py:120]: If we're going to keep it, I think we should call this `django_id` or `db_id`. It's internal to job-server db and should never be used externallly.

File: jobserver/api/jobs.py [hunk 3]
```diff
@@ -118,6 +118,8 @@ def post(self, request, *args, **kwargs):
...
+                    span.set_attribute("identifier", job.identifier)
```
Comment (bloodearnest) [inline jobserver/api/jobs.py:121]: I would call this "job_id"

**Review by bloodearnest — APPROVED** [review]

File: jobserver/api/jobs.py (hunk 3 above)
Comment (madwort) [reply jobserver/api/jobs.py:121]: willing to go to `job_identifier`? lots of potential confusion between id / identifier in this class...

**Pull request merged**